v5_MIGRATION_GUIDE.md

```diff
@@ -537,6 +542,27 @@ while (page.hasNextPage()) {
 }
 ```
 
+#### Accessing headers, data, rawResponse
+
+In v5, the process of accessing the raw response from an endpoint differs depending on whether the method in question is paginated. If it is paginated, the `.data` and `.rawResponse` fields are automatically provided:
+
+```ts
+// Paginated methods automatically return .rawResponse and .data:
+let users = await client.users.list({ per_page: 5, page: 1 });
+while (users.hasNextPage()) {
+    users = await users.getNextPage();
+}
```

Comment:
I do not think these 2 lines serve a purpose in this section, we should be able to remove them.